quaistats/quaistats.go

```diff
@@ -187,6 +187,14 @@ func (q *StatsQueue) Dequeue() interface{} {
 		return nil
 	}
 
+	return q.dequeue()
+}
+
+func (q *StatsQueue) enqueue(item interface{}) {
```

Comment:
Wouldn't be opposed to taking this out of a function. It's only one line and only used once, so it could be more readable than having to jump to definition. But either way is fine.

Reply:
Thought it made sense from a readability sake but fine to remove it. @gameofpointers What do you think?

Reply:
we dont need mutex on this ? If this function is only called from inside Enqueue, i dont think we need this